<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibrationParamReference</class>
 <widget class="QWidget" name="CalibrationParamReference">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>35</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelStepName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>170</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditParamName">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>5</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnChangeParam">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>2</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>…</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnScaleChangeParam">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>2</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>…</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditScaleParamName">
   <property name="geometry">
    <rect>
     <x>369</x>
     <y>5</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
